politeiawww/politeiawww.go

```diff
@@ -171,6 +175,7 @@ func (p *politeiawww) handleVersion(w http.ResponseWriter, r *http.Request) {
 		PubKey:       hex.EncodeToString(p.cfg.Identity.Key[:]),
 		TestNet:      p.cfg.TestNet,
 		Mode:         p.cfg.Mode,
+		Quiesce:      p.quiesce,
```

Comment:
You can't access shared state without holding the lock.  `p.cfg.TestNet` and `p.cfg.Mode` are static values, they don't change.  `p.quiesce` does.  Similar to the `setQuiesce`, make a function that holds the read lock and returns the quiesce value.  I would define these right under the `politeiawww` context in politeiawww.go.  If all the getters/setters are defined right with the context they get/set for then it keeps the code better organized since they'll likely be used in multiple files throughout politeiawww.